ios/FluentUI/Label/Label.swift

```diff
@@ -40,6 +41,13 @@ open class Label: UILabel, TokenizedControlInternal {
         }
     }
 
+    open override var font: UIFont! {
+        didSet {
+            _font = font
```

Comment:
i am confused why we need ivar here when probably app has its own _font?

Reply:
By "app has its own _font," do you mean UILabel's font (i.e. super.font)? We need the ivar so that we know when the token is overridden by a different font. The UILabel's font can't be set to nil (it can, but it really just resets it to the default font which isn't actually nil)

Reply:
sorry typo. Apple will have _font most likely for their font vairable in uilable. didSet shouldn't be actually setting the _font

Reply:
Hm, I don't see a native _font in UILabel